qln/msghandler.go

```diff
@@ -53,6 +47,28 @@ func (nd *LitNode) PeerHandler(msg lnutil.LitMsg, q *Qchan, peer *RemotePeer) er
 
 }
 
+// BroadcastMessage sends a chat message to all connected websocket clients
+func (nd *LitNode) BroadcastChatMessage(msg lnutil.LitMsg) error {
+	chat, ok := msg.(lnutil.ChatMsg)
+	if !ok {
+		return fmt.Errorf("can't cast to chat message")
+	}
+
+	// get the current set of channels and save them, they may be deleted by litrpc
+	// before the iteration is over
+	var channels []chan lnutil.ChatMsg
```

Comment:
golang channels can also close; if the websocket connection closes, the golang channel does as well, right?
what if nd.UserWsCons changes *after* iterating over it?  That seems more likely than if it changes during iteration.  And then you've got a reference to a closed go channels that you try to push messages to.
https://dave.cheney.net/2014/03/19/channel-axioms
sends to a closed channel panic / crash the program.